Mage.Verify/src/test/java/mage/verify/VerifyCardDataTest.java

```diff
@@ -1647,18 +1652,162 @@ private void checkCardCanBeCopied(Card card1) {
+                                    doFieldRecurse = false;
+                                    hasSpellAbilityField = true;
+                                } else if (field1.getName() == "meldsToCard") {
+                                    compareClassRecursive(((CardImpl) obj1).getMeldsToCard(), ((CardImpl) obj2).getMeldsToCard(), originalCard, msg + "::" + field1.getName(), maxDepth - 1, alreadyChecked, doRecurse);
+                                    doFieldRecurse = false;
+                                    hasMeldField = true;
+                                } else if (field1.getName() == "secondSideCard") {
+                                    compareClassRecursive(((CardImpl) obj1).getSecondCardFace(), ((CardImpl) obj2).getSecondCardFace(), originalCard, msg + "::" + field1.getName(), maxDepth - 1, alreadyChecked, doRecurse);
+                                    doFieldRecurse = false;
+                                    hasSecondSideCardField = true;
+                                }
+                            }
+                            if (class1 == AbilityImpl.class) {
+                                if (field1.getName() == "watchers") {
+                                    // Watchers are only used on initialization, they are not copied.
+                                    doFieldRecurse = false;
+                                    hasWatchersField = true;
+                                }
+                                if (field1.getName() == "modes") {
+                                    //compareClassRecursive(((AbilityImpl) obj1).getModes(), ((AbilityImpl) obj2).getModes(), originalCard, msg + "<" + obj1.getClass() + ">" + "::" + field1.getName(), maxDepth - 1);
+                                    compareClassRecursive(((AbilityImpl) obj1).getEffects(), ((AbilityImpl) obj2).getEffects(), originalCard, msg + "<" + obj1.getClass() + ">" + "::" + field1.getName(), maxDepth - 1, alreadyChecked, doRecurse);
+                                    doFieldRecurse = false;
+                                    hasModesField = true;
+                                }
+                            }
+                            if (doFieldRecurse) {
+                                compareClassRecursive(value1, value2, originalCard, msg + "<" + obj1.getClass() + ">" + "::" + field1.getName(), maxDepth - 1, alreadyChecked, doRecurse);
+                            }
+                        } catch (IllegalArgumentException | IllegalAccessException e) {
+                        }
+                        fieldIndex++;
+                    }
+
+                    // Do check that the expected special fields were encountered.
+                    // If those field are no relevant, or were renamed, please modify the relevant code block above.
```

Comment:
Please, add here hint how to fix it. Example:
how-to fix: broken class must have copy and copy constructor methods

Reply:
This particular comment is on "there are 5 special fields that can not be looped into. If those are renamed, the unit test need small rework."